### DISCUSSION TITLE
身份验证服务器目前处于宕机状态

### DISCUSSION THREAD

**Discussion by Tonmyn** [bs-community/blessing-skin-server#386] [Q&A]
我的服务器情况为 ：MC服务器和皮肤站部署在同一个机器上，具有公网IP，每隔三天就会变化。
我目前遇到的问题：在更换IP后，登陆服务器就会提示“身份验证服务器目前处于宕机状态。请稍后在试，抱歉！”，通过多次登陆才能进服务器。我不清楚这个问题出在皮肤站还是Yggdrasil API插件上面。

**Comment by Big-Cake-jpg** [comment]
请尝试清除你的 DNS 缓存
